mmv1/templates/terraform/decoders/firestore_database.go.erb

```diff
@@ -0,0 +1,13 @@
+config := meta.(*Config)
```

Comment:
This looks like it could potentially be replaced with a generic "flatten" function, but I'm not quite sure which. Perhaps `name_from_self_link` is enough? Do you know if there is special handling you need for this resource?

Reply:
Yup, that works. Thanks!